backend/app/crud/reservation.py

```
@@ -0,0 +1,186 @@
+from sqlalchemy.orm import Session
+from sqlalchemy import or_, and_, func
```

Ruff (0.11.9)

`sqlalchemy.and_` imported but unused

Remove unused import: `sqlalchemy.and_`

(F401)